.github/workflows/v2-merge-forward.yml

```diff
@@ -0,0 +1,23 @@
+name: 'v2-main merge forward'
+on:
+  schedule:
+    - cron: '0 9 * * 1,3,5' # 9AM UTC every M/W/F
```

Comment:
Why not every day?

Reply:
Let's start here. Initially I want to see how this works. I'll increase frequency once everything is set up and I see smooth movement.